package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I dont know if you want this package.json in here, but this runs composer update and starts the cypress project automatically if you type npm run e2e.

Of course, this requires both php,composer and cypress to be installed on the system